addons/cargo/functions/fnc_removeCargoItem.sqf

```diff
@@ -23,33 +23,47 @@ TRACE_3("params",_item,_vehicle,_amount);
 
 private _loaded = _vehicle getVariable [QGVAR(loaded), []];
 
-private _itemsRemoved = 0;
 private _addedSpace = 0;
+private _itemClass = _item;
 
-{
-    if (_itemsRemoved == _amount) exitWith {};
+private _continue = if (_item isEqualType objNull) then {
+    if (_item in _loaded) exitWith {false};
```

Comment:
Surely this should be `!(_item in _loaded)`

Reply:
🤦‍♂️